### ISSUE TITLE
Install pdns manager by script

### ISSUE THREAD

**Issue by dtoubelis** [loewexy/pdnsmanager#65]
I created an Ansible role to install pdnsmanger, it is pretty straightforward and works well. The role creates `config/config-user.php` file with database connection information, however, when I try to login, the UI goes in the infinite `Upgrade` -> `Login` cycle. I suspect that the problem is with initial user missing from the database. Is there a way to create this user programmatically with any kind of script or directly with SQL?

**Comment by loewexy** [comment]
Yeah that is definitely your problem. You can either have a look at the code of api/install.php and see how it is created there. Otherwise you can use curl to trigger the respective http call.
